packages/backend/src/tests/getTableTypes.ts

```diff
@@ -0,0 +1,9 @@
+export type TableTypes<Table> = {
```

Comment:
add some documentation (and example) of what this type does.

Reply:
Also not sure I like the naming as I think (but I'm not sure) that this returns columns, not tables 🤔